Package.swift

```diff
@@ -12,14 +12,22 @@ let package = Package(
     ],
     dependencies: [
         // Dependencies declare other packages that this package depends on.
-		.package(url: "https://github.com/ctreffs/SwiftSDL2.git", from: "1.1.0"),
-		.package(url: "https://github.com/fireblade-engine/ecs.git", from: "0.16.0")
+        .package(name: "SDL2", url: "https://github.com/ctreffs/SwiftSDL2.git", from: "1.1.0"),
+        .package(name: "FirebladeECS", url: "https://github.com/fireblade-engine/ecs.git", from: "0.17.4"),
+        .package(name: "FirebladeMath", url: "https://github.com/fireblade-engine/math.git", from: "0.9.1")
     ],
     targets: [
         // Targets are the basic building blocks of a package. A target can define a module or a test suite.
         // Targets can depend on other targets in this package, and on products in packages which this package depends on.
         .target(
             name: "FirebladeECSDemo",
-            dependencies: ["FirebladeECS", "SDL2"])
+            dependencies: ["FirebladeECS", "SDL2"]),
+        .target(
+            name: "Asteroids",
+            dependencies: ["FirebladeECS", "SDL2", "FirebladeMath", "Library"],
+            exclude: ["Resources/source.txt"],
+            resources: [.copy("Resources/asteroid.wav"), .copy("Resources/ship.wav"), .copy("Resources/shoot.wav")]),
+        .target(name: "Library",
```

Comment:
I would like to propose you rename `Library` to `AsteroidsGameLibrary` or just `Game2DLibrary` or something a little more specific than just `Library`. This is no necessity but it makes the purpose of the library a lot clearer and since a library is something you want to be reused by others this will help.
If you just want it to be used internally then you could name it something like `AsteroidsCommon` or `Utils` or something similar to that.

Reply:
Sure. AsteroidsGameLibrary is the name for it from now on.